### PR TITLE
doc: Broadcast Stream operator, #25468

### PR DESCRIPTION
**Pull request by patriknw**
Refs #25468

### CODE REVIEW

File: akka-docs/src/main/paradox/stream/operators/Broadcast.md [hunk 1]
```diff
@@ -4,10 +4,32 @@ Emit each incoming element each of `n` outputs.
...
+## Signature
+
+@apidoc[akka.stream.*.Broadcast]
+
 ## Description
 
 Emit each incoming element each of `n` outputs.
 
+## Example
+
```
Comment (johanandren) [inline akka-docs/src/main/paradox/stream/operators/Broadcast.md]: Some friendly text explaining the sample would be nice. Perhaps also mention the fact that the downstreams are not running in parallel but on the same island/actor.

Something about the scala Tuple3 in the Java API would also be good.

**Review by johanandren — APPROVED** [review]
LGTM but more chat would be good

File: akka-docs/src/main/paradox/stream/operators/Broadcast.md (hunk 1 above)
Comment (patriknw) [reply akka-docs/src/main/paradox/stream/operators/Broadcast.md]: Added intro text.

"not running in parallel" was already mentioned, and there is even a separate example illustrating that

Tuple3 is not Scala. It's `import akka.japi.tuple.Tuple3`, which is shown in the included imports

**Review by johanandren — APPROVED** [review]

**Review by raboof — APPROVED** [review]

**Comment by phiSgr** [comment]
It looks like the broadcast-async example is not actually parallelizing the execution.
I have [a reproducer](https://github.com/phiSgr/akka-streams-broadcast).

**Pull request merged**